src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -152,9 +149,25 @@ protected virtual CodeActionPriority ComputePriority()
         /// </summary>
         public virtual ImmutableArray<string> Tags => ImmutableArray<string>.Empty;
 
-        internal virtual ImmutableArray<CodeAction> NestedCodeActions
+        /// <summary>
+        /// Child actions contained within this <see cref="CodeAction"/>.  Can be presented in a host to provide more
+        /// potential solution actions to a particular problem.  To create a <see cref="CodeAction"/> with nested
+        /// actions, use <see cref="Create(string, ImmutableArray{CodeAction}, bool)"/>.
+        /// </summary>
+        public virtual ImmutableArray<CodeAction> NestedActions
```

Comment:
📝 This rename is going to break a bunch of existing test code. Recommend either not renaming it, or including a private property with the original name that defers to the new property so existing test code can access it through that property.

Reply:
Sure.  I don't mind that.

Reply:
Added back and filed issue on SDK to remove reflection.